### PR TITLE
Functional components will appear in the vue devtools 5.x version

### PR DESCRIPTION
**Pull request by bailnl**
The following is a screenshot of vue devtools

image

### CODE REVIEW

File: src/v2/guide/render-function.md [hunk 1]
```diff
@@ -544,7 +544,7 @@ Everything the component needs is passed through `context`, which is an object c
 
 After adding `functional: true`, updating the render function of our anchored heading component would require adding the `context` argument, updating `this.$slots.default` to `context.children`, then updating `this.level` to `context.props.level`.
 
-Since functional components are just functions, they're much cheaper to render. However, the lack of a persistent instance means they won't show up in the [Vue devtools](https://github.com/vuejs/vue-devtools) component tree.
+functional components are just functions, they're much cheaper to render. now they can show up in the [Vue devtools v5+](https://github.com/vuejs/vue-devtools) component tree, but the previous version won't.
```
Comment (phanan) [inline src/v2/guide/render-function.md]: I believe it's way better to just keep the original paragraph without the "However" sentence.

Comment (Justineo) [inline src/v2/guide/render-function.md]:
```suggestion
Since functional components are just functions, they're much cheaper to render.
```

**Comment by bailnl** [comment]
@Justineo @Justineo 
at first i thought so too. 
https://github.com/vuejs/vuejs.org/commit/8eba802a11d76ad84ba4671b9d48090521ea1275

**Comment by sdras** [comment]
This looks great now thanks!

**Pull request merged**